custom_components/adaptive_lighting/switch.py

```diff
@@ -770,6 +821,17 @@ def __init__(
         self._name = data[CONF_NAME]
         self._interval = data[CONF_INTERVAL]
         self._lights = data[CONF_LIGHTS]
+        self._manual_lights = {}
+        all_lights = _expand_light_groups(self.hass, self._lights)
+        for light in all_lights:
+            self._manual_lights[light] = {"timer": 0}
+
+        for light in data[CONF_LIGHTS]:
+            if not hasattr(self.turn_on_off_listener, "_switches"):
+                self.turn_on_off_listener._switches = {}
+            self.turn_on_off_listener._switches[
+                light
+            ] = self  # light should never be in multiple switches anyway.
```

Comment:
I have personally have had the same light in multiple switches, and @sizors also reported to be doing that here https://github.com/basnijholt/adaptive-lighting/issues/78#issuecomment-1484502712

Reply:
Just realized that this `turn_on_off_listener._switches` is never used. Please remove.